packages/react-charting/src/components/TreeChart/TreeChart.base.tsx

```diff
@@ -0,0 +1,293 @@
+/* eslint-disable dot-notation */
+/* eslint-disable one-var */
+/* eslint-disable @fluentui/max-len */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-console */
+/* eslint-disable prefer-const */
+import * as React from 'react';
+import { createRef } from 'react';
+import { select, Selection } from 'd3-selection';
+import { hierarchy, tree } from 'd3-hierarchy';
+
+import { ITreeProps, ITreeState, ITreeDataStructure, IDataStructure } from '../../index';
```

Comment:
rename IDataStructure to something better

Reply:
fixed (renamed to IChartDataPoint)